presidio-analyzer/analyzer/entity_recognizer.py

```diff
@@ -276,12 +276,12 @@ def find_index_of_match_token(word, start, tokens, tokens_indices):
                              "is expected to be found")
         return i
 
-    def __extract_surrounding_words(self, nlp_artifacts, word, start):
-        """ Extracts words surronding another given word.
+    def __extract_context(self, nlp_artifacts, word, start):
```

Comment:
function name has changed

Reply:
missed that, thanks for catching it!

Reply:
Please fix :)

Reply:
Sorry for the slow update, I'm still learning about Git and thought the PR review meant it was fixed. Fixed and committed now :)

Reply:
Awesome. Thanks so much!